integration-tests/ssr/test-output.js

```diff
@@ -42,6 +42,15 @@
       )
     )
 
+    // Fetch once to trigger re-compilation.
+    await fetch(`${devSiteBasePath}/${path}`)
+
+    // Then wait for 6 seconds to ensure it's ready to go.
+    // Otherwise, tests are flaky depending on the speed of the testing machine.
+    await new Promise(resolve => {
+      setTimeout(() => resolve(), 6000)
+    })
+
```

Comment:
I don't really understand what this does 😬 Is this to make sure it got rendered?

Reply:
yes — because ssr bundling is lazy & because we don't wait for it to finish necessarily — in tests when we _really_ do want ssr to happen, we have to wait